plugins/app-backend/dev/index.ts

```diff
@@ -14,19 +14,8 @@
  * limitations under the License.
  */
 
-import { getRootLogger } from '@backstage/backend-common';
-import { startStandaloneServer } from './service/standaloneServer';
+import { createBackend } from '@backstage/backend-defaults';
 
-const logger = getRootLogger();
-
-startStandaloneServer({ logger }).catch(err => {
-  logger.error(err);
-  process.exit(1);
-});
-
-process.on('SIGINT', () => {
-  logger.info('CTRL+C pressed; exiting.');
-  process.exit(0);
-});
-
-module.hot?.accept();
+const backend = createBackend();
+backend.add(import('../src/alpha'));
```

Comment:
How can I configure the app package name?

Reply:
Yep, exactly. I realize you might even be able to use `mockServices.config` to set it up, the linter should be alright with a dev dep here

Reply:
Done 🙌🏻